### ISSUE TITLE
In-Game VoIP Not Functional

### ISSUE THREAD

**Issue by ufgkirk** [DomiStyle/docker-eldewrito#17]
Hi,

I currently have the following in my config, I believe this is default:
```
VoIP.Enabled "1"
VoIP.PTT_Enabled "1"
VoIP.MicrophoneID ""
VoIP.EchoCancelation "1"
VoIP.AGC "1"
VoIP.NoiseSupress "1"
VoIP.PTTSoundEnabled "1"
VoIP.SpeakingPlayerOnHUD "0"
VoIP.StereoVoice "1"
Server.SignalServerPort "11777"
```

I see the Docker proxy listening on port 11777, and it's open in my firewall:
`LISTEN     0      128         :::11777                   :::*                   users:(("docker-proxy",pid=30898,fd=4))`
```
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpts:11774:11777
ACCEPT     udp  --  0.0.0.0/0            0.0.0.0/0            udp dpts:11774:11777
```

However, the VoIP is not working in the game for any players.  When you press caps lock, I see the icon illuminate like you're talking, but the VoIP traffic isn't relayed.  What could possibly be causing this?

**Comment by DomiStyle** [comment]
You could try the following:
* Remove your existing dewrito_prefs.cfg and recreate the container to let the server generate a safe default that is known to work
* If the players are connecting from outside of your own network make sure you have port forwarded [the VoIP port](https://github.com/DomiStyle/docker-eldewrito#ports) on both your host and your router/modem
* Use host network instead of forwarding ports manually

**Comment by ufgkirk** [comment]
Hi DomiStyle,

Looks like we got this resolved, I let the Docker generate a default configuration, and I noticed a few differences between that and the configuration that I originally generated on Windows.

`Game.Discord.Enable "0"`
`UPnP.Enabled "0"`

I believe the Discord one is just for the overlay, but this isn't needed on a server, I turned that off.

UPnP.Enabled was set to 1 in our config.  I think this was the issue, although I'm not entirely sure why it would break things even if unused.  However, we aren't using UPnP in our environment.  This is a dedicated server with direct access to the Internet.

It looks like changing those two settings has corrected our VoIP issues. :)
